requirements.txt

```diff
@@ -3,3 +3,4 @@ colorclass
 requests
 PyYAML
 enum34
+xdg
```

Comment:
This module looks simple enough, but is it really worth adding another external dependency for what is essentially a one-liner?

Reply:
Personally I'd opt to use the module, if they add more OS' etc we get it for free and the dep is really small in the scheme of things (all the dub dependencies of requests, pyaml, etc)

Reply:
I don't feel strongly about this. It is a super small module and adding it could result in less maintenance and better defaults across different OS's over time. @Dorthu deferring to you on this one.

Reply:
Yeah, you've convinced me, this is fine